src/ingest.py

```diff
@@ -162,12 +201,17 @@ def main():
 
             # Ingest if we have resources
             if resources:
+
+                if not is_lfa:
+                    record_groups(resources)
+
                 logger.info("Ingesting %s", resources)
                 refs = None
                 try:
                     refs = ingester.run(resources)
                 except Exception:
                     logger.exception("Error while ingesting %s", resources)
+                    continue
```

Comment:
Why is this needed with this commit?

Reply:
This is actually a separate fix that prevents visit definition (and Rucio registration) from trying to run if an exception occurred in ingest.  I'll split it out into a separate commit.